README.md

```diff
@@ -290,8 +290,8 @@ settings in the environment file:
 - IPS: A space separated list of IP addresses. Used to generate [ALLOWED_HOSTS](
   https://docs.djangoproject.com/en/3.0/ref/settings/#allowed-hosts).
 - FQDN: The server’s fully qualified domain name.
-- MAX_TASK_RESULTS: The maximum number of task results to keep before overwriting old
-  results. Defaults to 100,000.
+- MAX_DISK_USAGE: The maximum disk usage percentage allowed before overwriting old
```

Comment:
Should we add a note that there will be some difference with the usage reported in Linux?